mlflow/tracking/client.py

```diff
@@ -847,6 +848,40 @@ def set_trace_tag(self, request_id: str, key: str, value: str):
         # If the trace is not active, try to set the tag on the trace in the backend
         self._tracking_client.set_trace_tag(request_id, key, value)
 
+    def delete_trace_tag(self, request_id: str, key: str) -> None:
+        """
+        Delete a tag on the trace with the given trace ID.
+
+        The trace can be an active one or the one that has already ended and recorded in the
+        backend. Below is an example of deleting a tag on an active trace. You can replace the
+        ``request_id`` parameter to setting a tag on an already ended trace.
```

Comment:
```suggestion
        backend. Below is an example of deleting a tag on an active trace. You can replace the
        ``request_id`` parameter to deleting a tag on an already ended trace.
```

Should we use `deleting` here?

Reply:
Should we use `delete` instead of `deleting`?